broker-daemon/utils/migrate-store.spec.js

```diff
@@ -1,117 +1,201 @@
-const { sinon, expect } = require('test/test-helper')
+const path = require('path')
+const {
+  sinon,
+  expect,
+  rewire
+} = require('test/test-helper')
 
-const migrateStore = require('./migrate-store')
+const migrateStore = rewire(path.resolve(__dirname, './migrate-store'))
```

Comment:
did you base this off another branch? Including `migrateStore` in this PR seems quite strange.

Reply:
@treygriffith im not sure why the dif was weird